resources/gemini-1b/extra_nodes.tf

```diff
@@ -0,0 +1,128 @@
+resource "digitalocean_droplet" "gemini-1b-extra" {
+  count = var.extra-droplets
+  image  = "ubuntu-20-04-x64"
```

Comment:
why ubuntu 20.04 and not 22.04?

Reply:
22.04 is also LTE, let's use 22.04 going forward by default, nothing wrong with using it for existing instances.